src/modules/tracker.js

```diff
@@ -1268,6 +1272,8 @@ class Tracker {
         analyticsTags,
         seedItemIds,
         resultCount = result_count || items?.length || 0,
+        slCampaignId,
+        slCampaignOwner,
```

Comment:
I'm not sure if we need this here 🤔 

I believe we pass `slCampaignId` and `slCampaignOwner` via the `items` array. That seems to be how home24 has it implemented for Search/Browse based on what I see in their beacon. cc @esezen 

<img width="823" height="565" alt="Screenshot 2025-08-27 at 5 10 37 PM" src="https://github.com/user-attachments/assets/24df6a9c-57a6-4089-a9a5-d30133ad5452" />

Reply:
Hey @jjl014 , just verified this. Thanks for clarifying! Oh yeah it seems that's already supported out of the box for all items schemas. No changes required on either behavioral endpoint or the beacons here 👍.

Will close this PR since it's no longer required.